content/reference/drops/context_drop.md

```diff
@@ -0,0 +1,14 @@
+# Context Drop
```

Comment:
Well, it's used in very specific usecase, but exposed publicly, I think good to have it documented, even just for consistency.